src/rpc/server.cpp

```diff
@@ -484,13 +492,19 @@ std::vector<std::string> CRPCTable::listCommands() const
     return commandList;
 }
 
-UniValue CRPCTable::dumpArgMap() const
+UniValue CRPCTable::dumpArgMap(const JSONRPCRequest& args_request) const
 {
+    JSONRPCRequest request(args_request);
+    request.mode = JSONRPCRequest::GET_ARGS;
+    request.params = UniValue();
+
     UniValue ret{UniValue::VARR};
     for (const auto& cmd : mapCommands) {
-        for (const auto& c : cmd.second) {
-            const auto help = RpcMethodFnType(c->unique_id)();
```

Comment:
Holy crap, I'm happy to get rid of this.

Reply:
I was just testing our review process, obviously :sweat_smile: